eventarc/v1/python/google-cloud-eventarc/v1/eventarc_client_google_channel_config_get.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+# [START eventarc_v1_eventarc_googlechannelconfig_get]
+from google.api_core import exceptions as core_exceptions
+from google.cloud import eventarc_v1
+
+
+def get_google_channel_config(
+    project_id: str,
+    location: str,
+) -> None:
+    """
+    Retrieves the GoogleChannelConfig for a given project and location.
+
+    The GoogleChannelConfig stores custom settings respected by Eventarc first-party
+    triggers in the matching region. Once configured, first-party event data will
+    be protected using the specified custom managed encryption key instead of
```

Comment:
Change to "Once configured, first-party event data is protected using the specified customer-managed encryption key instead of Google-managed encryption keys." (I've noted that the docs need to be updated as well)